File: PWEB2-Front/mdb-angular-free/src/app/models/Aluno.ts
import {Matricula} from '../models/Matricula';
import {Projetos} from '../models/Projetos';
import {Bolsas} from '../models/Bolsas';


export class Aluno {
    id: string;
    nome:string;
    matriculas: Matricula[]; // config matricula
    projetos:Projetos[] // Config Modelo
    bolsas:Bolsas[]; // Config Modelo

}

File: Api-Gateway/src/main/java/com/pweb2/appifpb/model/Aluno.java
package com.pweb2.appifpb.model;

import lombok.Data;
import lombok.ToString;

import java.io.Serializable;
import java.util.List;

@ToString
@Data
public class Aluno implements Serializable {

    public String id;
    public String nome;

    public List<Projeto> projetos;

    public List<Bolsa> bolsas;

    public List<Matricula> matriculas;
}
